tests/lib.rs

```diff
@@ -593,6 +593,48 @@ fn map_into() {
     assert!(copy[&2].contains(&"c"));
 }
 
+#[test]
+fn keys() {
+    let (r, mut w) = evmap::new();
+    w.insert(1, "a");
+    w.insert(1, "b");
+    w.insert(2, "c");
+    w.refresh();
+    w.insert(1, "x");
+
+    let mut keys = r
```

Comment:
Looking at this usage ... this doesn't look very helpful ...
I wonder if it would be better to have `r.keys_blocking()` or something so that the user doesn't have to deal with flattening out the `Some()`s and is fairly aware of the blocking nature?

Reply:
I'd really rather avoid that. The use-case for `evmap` is generally that you're _first_ going to get access to the read ref (through `read()`), and only if that succeeds do you do the operation. I'd like to not shorthand going through `read`, because it'd basically mean we should duplicate all the methods.